### DISCUSSION TITLE
GST Data Sharing  - What and How?

### DISCUSSION THREAD

**Discussion by SanjanaIL** [ONDC-Official/ONDC-FIS-Specifications#39] [General]
![image](https://github.com/ONDC-Official/ONDC-FIS-Specifications/assets/142986416/7938fc41-634a-4c6f-a937-2fdcc8c74b1c)

The list of invoices from GSP are shared with all lenders or selected lenders? 
If the latter, how are we deciding the set of lenders to show a particular user's data with?

![image](https://github.com/ONDC-Official/ONDC-FIS-Specifications/assets/142986416/f23fa473-6408-4149-9723-289ae01d0d09)
Is this form going to be present on the Buyer app or the Seller App?

**Comment by 92shreyansh** [comment]
The Buyer App as per there user experience can determine if they want to show  static offers and let the user make a choice of lender based on those, or in case to visualize actual offers based on the information provided the buyer app will broadcast the user data to all the lenders. 

All forms re to be based vailabe by the Seller App the buyer app will fetch it and display it as it on there UI or process it in there system and use a UX of there own. In both cases the form should be submitted to the URI provided as part of the form submit action
